tools/ci/Dockerfile.prod.template

```diff
@@ -0,0 +1,17 @@
+# To rebuild from this file for a given server release, say 3.4:
+#   0. $ tools/ci/generate-dockerfiles
+#   1. $ sudo docker build tools/ci/images/$VERSION/ --tag zulip/ci-prod-upg:$VERSION
```

Comment:
ci-prod-upg has a lot of abbreviation in it; we generally avoid abbreviations in the project, and I'd prefer e.g. `ci-production-upgrade` as a key.